modules/nf-core/learnmsa/align/main.nf

```diff
@@ -0,0 +1,47 @@
+
+process LEARNMSA_ALIGN {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "bioconda::learnmsa=1.3.2"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/learnmsa:1.3.2--pyhdfd78af_0':
+        'biocontainers/learnmsa:1.3.2--pyhdfd78af_0' }"
+
+    input:
+    tuple val(meta),  path(fasta)
+
+    output:
+    tuple val(meta), path("*.aln"), emit: alignment
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    learnMSA \\
+        $args \\
+        -i $fasta \\
+        -o ${prefix}.aln
```

Comment:
just checking: this tool doesn't take an avaialble cpu number? Couldn't find docs listing all params

Reply:
In that case and if it is single threaded, I would change the ressource label to single

Reply:
Not as far as I know - it should ideally run on GPUs and, per default, will use all the available ones. Apparently, one can specify the name of the GPU devices, but not as a number, just as IDs - and this should probably go in $args.